gensim/models/phrases.py

```diff
@@ -210,10 +210,12 @@ def load(cls, *args, **kwargs):
         # update older models
         # if value in phrasegrams dict is a tuple, load only the scores.
         try:
-            if isinstance(list(model.__dict__['phrasegrams'].values())[0], tuple):
-                model.__dict__['phrasegrams'].update((k, v[1]) for k, v in model.__dict__['phrasegrams'].items())
+            for components, scores in model.__dict__['phrasegrams'].items():
```

Comment:
Why this strange `__dict__` access? Why not use the pattern I showed in my last review?